java-runtime-builder-app/src/main/java/com/google/cloud/runtimes/builder/config/domain/BuildContext.java

```diff
@@ -152,9 +155,11 @@ private void writeDockerFile() throws IOException {
           + "from the root of your sources to continue.");
     }
 
-    if (Files.exists(workspaceDir.resolve(EXISITING_DOCKER_DIR + DOCKERFILE_NAME))) {
-      throw new IllegalStateException("There is an unexpected Dockerfile in " + EXISITING_DOCKER_DIR
-      + ". Please remove it to continue.");
+    for (String dir : EXISITING_DOCKER_DIRS) {
```

Comment:
Can we merge this with the block above? The message can be what we have above, slightly modified to point to the path.